xhr/client/src/main/scala/endpoints4s/xhr/EndpointsSettings.scala

```diff
@@ -3,38 +3,45 @@ package endpoints4s.xhr
 import endpoints4s.Hashing
 
 import scala.annotation.nowarn
+import scala.concurrent.duration.{FiniteDuration}
 
 /** Settings for XHR interpreter.
   * @param baseUri Base of the URI of the service that implements the endpoints, can be absolute or relative (e.g. "http://foo.com" or "/bar")
   */
-final class EndpointsSettings private (val baseUri: Option[String]) extends Serializable {
+final class EndpointsSettings private (val baseUri: Option[String], val timeout: Option[FiniteDuration]) extends Serializable {
 
   override def toString =
-    s"EndpointsSettings($baseUri)"
+    s"EndpointsSettings($baseUri, $timeout)"
 
   override def equals(other: Any): Boolean =
     other match {
       case that: EndpointsSettings =>
-        baseUri == that.baseUri
+        baseUri == that.baseUri && timeout == that.timeout
       case _ => false
     }
 
-  override def hashCode(): Int = Hashing.hash(baseUri)
+  override def hashCode(): Int = Hashing.hash(baseUri, timeout)
 
   @nowarn("cat=unused")
```

Comment:
It seems this annotation is not necessary?

Reply:
The annotation was in the code already. I don't know, which warning it solves.

Should I remove it?